static/src/js/components/AdminPreferencesMetrics/AdminPreferencesMetricsList.scss

```diff
@@ -0,0 +1,28 @@
+.admin-preferences-metrics-list {
```

Comment:
Double check if any of the styles in this file can be added with bootstrap instead of custom css.